src/python/k4a/src/k4a/_bindings/k4a.py

```diff
@@ -0,0 +1,525 @@
+k4a_device_get_version.argtypes = (_DeviceHandle, _ctypes.POINTER(HardwareVersion))
+
+
+#K4A_EXPORT k4a_status_t k4a_device_get_color_control_capabilities(k4a_device_t device_handle,
+#                                                                  k4a_color_control_command_t command,
+#                                                                  bool *supports_auto,
+#                                                                  int32_t *min_value,
+#                                                                  int32_t *max_value,
+#                                                                  int32_t *step_value,
+#                                                                  int32_t *default_value,
+#                                                                  k4a_color_control_mode_t *default_mode);
+k4a_device_get_color_control_capabilities = _k4a_lib.k4a_device_get_color_control_capabilities
+k4a_device_get_color_control_capabilities.restype = EStatus
+k4a_device_get_color_control_capabilities.argtypes = (
+    _DeviceHandle, _ctypes.c_int, _ctypes.POINTER(_ctypes.c_bool),
+    _ctypes.POINTER(_ctypes.c_int32), _ctypes.POINTER(_ctypes.c_int32),
+    _ctypes.POINTER(_ctypes.c_int32), _ctypes.POINTER(_ctypes.c_int32),
+    _ctypes.POINTER(_ctypes.c_int) )
+
+
+#K4A_EXPORT k4a_status_t k4a_device_get_color_control(k4a_device_t device_handle,
+#                                                     k4a_color_control_command_t command,
+#                                                     k4a_color_control_mode_t *mode,
+#                                                     int32_t *value);
+k4a_device_get_color_control = _k4a_lib.k4a_device_get_color_control
+k4a_device_get_color_control.restype = EStatus
+k4a_device_get_color_control.argtypes = (_DeviceHandle, _ctypes.c_int, _ctypes.POINTER(_ctypes.c_int), _ctypes.POINTER(_ctypes.c_int32))
+
+
+#K4A_EXPORT k4a_status_t k4a_device_set_color_control(k4a_device_t device_handle,
+#                                                     k4a_color_control_command_t command,
+#                                                     k4a_color_control_mode_t mode,
+#                                                     int32_t value);
+k4a_device_set_color_control = _k4a_lib.k4a_device_set_color_control
+k4a_device_set_color_control.restype = EStatus
+k4a_device_set_color_control.argtypes = (_DeviceHandle, _ctypes.c_int, _ctypes.c_int, _ctypes.c_int32)
+
+
+#K4A_EXPORT k4a_buffer_result_t k4a_device_get_raw_calibration(k4a_device_t device_handle,
+#                                                              uint8_t *data,
+#                                                              size_t *data_size);
+k4a_device_get_raw_calibration = _k4a_lib.k4a_device_get_raw_calibration
+k4a_device_get_raw_calibration.restype = EBufferStatus
+k4a_device_get_raw_calibration.argtypes = (_DeviceHandle, _ctypes.POINTER(_ctypes.c_uint8), _ctypes.POINTER(_ctypes.c_ulonglong))
```

Comment:
there is a ctpyes c_size_t
That might be more extensible if we want to support both 32 and 64 bit

Reply:
Changed to c_size_t.